action.yml

```diff
@@ -22,3 +22,15 @@ runs:
           exit 1
         fi
       shell: bash
+    - name: Check PR description for issue URL
+      shell: bash
```

Comment:
This should be optional as there are also teams that use Jira instead of GitHub issues.

Reply:
Added optional input parameter.